frontend/src/utils/documentListHelper.js

```diff
@@ -142,7 +141,8 @@ const DLmapStateToProps = (state, props) => {
     parentSelected: parentSelector(state, parentTableId),
     modal: state.windowHandler.modal,
     rawModalVisible: state.windowHandler.rawModal.visible,
-    filters: state.filters,
+    filters: windowId && viewId && state.filters ? state.filters[filterId] : {},
```

Comment:
Hmm. Not sure. This is faster

Reply:
Only in case `filters[filterId]` exists. Otherwise we'll get undefined. I think it'd be best to have a cached selector here as filters might be a complicated object and hit us with unnecessary re-renders.